lms/djangoapps/verify_student/views.py

```diff
@@ -594,6 +613,10 @@ def _requirements(self, display_steps, is_active):
             self.WEBCAM_REQ: False,
         }
 
+        # Remove the account activation requirement if disabled via waffle
+        if DISABLE_ACCOUNT_ACTIVATION_REQUIREMENT_FLAG:
```

Comment:
I'm not familiar with waffle, but i'm curious why previously in the file you call waffle.switch_is_active and here you reference the flag directly? Is there a difference?